arch/risc-v/src/mpfs/Make.defs

```diff
@@ -112,4 +113,6 @@ endif
 
 ifeq (${CONFIG_MPFS_BOOTLOADER},y)
 CHIP_CSRCS += mpfs_cache.c
+else
+CHIP_CSRCS += mpfs_scratch.c
```

Comment:
This will be fixed, there is no reason to omit the module (original idea was that the module would be omitted in bl build, but there is no reason).